bundle/manifests/customized-user-remediation.clusterserviceversion.yaml

```diff
@@ -70,7 +76,19 @@ spec:
         displayName: Template
         path: template
       version: v1alpha1
-  description: TBD
+  description: |
+    ### Introduction
+    Customized User Remediation (CUR) works with Node Health Check to remediate unhealthy nodes by running a user customized script on the unhealthy node.
+    This operator does not preform any fencing on the remediated node so users should be careful with the scripts that are being used.
+
+    ### Compatibility
+    CUR is one of the remediator operators by [Medik8s](https://www.medik8s.io/remediation/remediation/),
+    such as [Self Node Remediation](https://github.com/medik8s/self-node-remediation) and [Machine Deletion Remediation](https://github.com/medik8s/machine-deletion-remediation),
+    that were designed to run with the Node HealthCheck Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator) which detects an unhealthy node and creates remediation CR.
+    It is recommended to use FAR with NHC for an easier and smoother experience by fully automating the remediation process, but it can be used as a standalone remediator for the more experienced user.
```

Comment:
I assume you mean the description, so yes

Reply:
Why should it be included in the CUR CSV's description? 
It is nice to mention [other alternatives](https://github.com/medik8s/customized-user-remediation/pull/14#discussion_r1543004260) for remediation with NHC, but I am unsure how relevant it is for CUR user (and to put it in the CUR description) how FAR and NHC is a good combination. 
IMHO this description should mostly focus on CUR, and less on other remediators (and their combinations).

Reply:
At CUR's current maturity level I think it's good enough. It basiclly follows the same template of [FAR](https://github.com/medik8s/fence-agents-remediation/blob/main/config/manifests/bases/fence-agents-remediation.clusterserviceversion.yaml#L107-L111).
I think that perfecting CUR documentation will make much more sense in case it'll reach a much higher maturity level, ATM I fine with settling for sub optimal documentation as long as it's not grossly misleading or incorrect.